common/include/vecmat.h

```diff
@@ -163,7 +163,7 @@ class vm_distance_squared
 	}
 	static constexpr vm_distance_squared maximum_value()
 	{
-		return vm_distance_squared{0x7fffffffffffffff};
+		return vm_distance_squared{static_cast<fix64>(0x7fffffffffffffff)};
```

Comment:
Yes that works fine.

It probably isn't using the deleted constructor, just noting that it has been deleted. The other constructor being considered would probably be the implicit move or implicit copy constructor, which would make Clang think it's ambiguous (even though `vm_distance_squared` is a wrapped `fix64`). I don't really know, Clang has some pretty zealous warnings.